### DISCUSSION TITLE
☓度数分布→○ヒストグラム（"histogram" の訳語が「ヒストグラム」と「度数分布」でばらついている）

### DISCUSSION THREAD

**Discussion by pato-taityo** [pgsql-jp/jpug-doc#2437] [NGとする語]
## 問題点

"histogram" の訳語が「ヒストグラム」と「度数分布」でばらついている。

## 背景

https://www.postgresql.org/docs/13/view-pg-stats.html
https://www.postgresql.jp/document/13/html/view-pg-stats.html
"histogram" を「ヒストグラム」と訳しています。

https://www.postgresql.org/docs/13/row-estimation-examples.html
https://www.postgresql.jp/document/13/html/row-estimation-examples.html
"histogram" を「度数分布」と訳しています。

## 解決方法

"histogram" の訳語を「ヒストグラム」と「度数分布」のいずれかで統一するのはいかがでしょうか。

## 注意点

https://www.stat.go.jp/naruhodo/4_graph/shokyu/histogram.html
この記事を含めいくつか目を通したところ、「ヒストグラム」はグラフを指しているようです。「度数分布」と表現する方がより適切に思えます。
ただ、本家（英語版）の表記が "histogram" となっているのが悩ましいところです…。

**Comment by KenichiroTanaka** [comment]
すみません、反応が遅くなりました。
どちらかで統一すべき、という点はご指摘の通りです。

あとはどちらに統一すべきか、ですが、

https://docs.oracle.com/cd/E57425_01/121/TGSQL/tgsql_histo.htm

Oracleではヒストグラムと表現されており、データベースの用語としてはこちらで良いように思います。

他にご意見ある方いらっしゃいますでしょうか。

**Comment by pato-taityo** [comment]
検討していただき、ありがとうございます。
起票者として、他のデータベース製品での表記についても確認してみました。

SQL Server は「ヒストグラム」（英語版では Histogram）としていました。
https://docs.microsoft.com/ja-jp/sql/relational-databases/statistics/statistics?view=sql-server-ver15

Db2 for z/OS は「ヒストグラム統計」としていました。
https://www.ibm.com/docs/ja/db2-for-zos/12?topic=statistics-collecting-histogram

逆に「度数分布」という用語がデータベース製品 (Db2, SQL Server, Oracle Database, MySQL) の
マニュアルで使われているかを Google 検索で調べてみました。
統計情報を指す用語として使われている箇所は見つけられませんでした。

データベースの用語としては「ヒストグラム」と表現するのが一般的と思われること、
英語版では "histogram" としていることを踏まえると、
「ヒストグラム」と統一するのが適切と考えました。

**Comment by KenichiroTanaka** [comment]
ありがとうございます。
他に異論もなさそうですので、ヒストグラムで進めていきましょう。
どなたかPRを作っていただくことは可能でしょうか？

**Comment by pato-taityo** [comment]
起票者としてPRを作成してみました。

https://github.com/pgsql-jp/jpug-doc/wiki/JPUG-DOC%E3%81%AE%E7%BF%BB%E8%A8%B3%E6%B4%BB%E5%8B%95%E3%81%B8%E3%81%AE%E5%8F%82%E5%8A%A0%E6%96%B9%E6%B3%95(%E5%88%9D%E3%82%81%E3%81%A6%E3%81%AE%E6%96%B9%E5%90%91%E3%81%91)
こちらを参考に作業を進めましたが、以下はできていません。
- MLへの加入及び査読依頼
- 作業ファイルの宣言
- コンパイル確認

必須の作業があればご教示ください。
コンパイルはこの後試してみようと思います…。

よろしくお願いいたします。

**Comment by KenichiroTanaka** [comment]
ありがとうございます！

記事が少し古くPRの一覧でタイトルの横（レビュー待ちのラベルの左）にあるチェックが緑色になっていれば
コンパイル確認は不要です（とはいえ、手元でコンパイルができる環境があると便利です）。

PRは内容を確認します。

**Comment by KenichiroTanaka** [comment]
PRを確認しました。
問題ありませんのでマージしました。

ご報告及びPRの作成ありがとうございました。
ISSUESはクローズしますがPRの出し方や翻訳に関して分からないことがありましたら
本ISSUESを更新していただくかslackやメーリングリストにご連絡ください。

**Comment by pato-taityo** [comment]
丁寧な対応をありがとうございました。

ドキュメント翻訳担当リスト14.0の貢献内容に追記させていただきました(PR2171)。
前回がおんぶにだっこで掲載していただくのが恐縮するような内容だったので…。

今後ともよろしくお願いいたします。
